schema/suggested_genes.js

```diff
@@ -12,10 +12,10 @@ const SuggestedGeneType = new GraphQLObjectType({
 })
 
 const SUGGESTED_GENES_JSON = "https://s3.amazonaws.com/eigen-production/json/eigen_categories.json"
-const SuggestedGenes = {
+const SuggestedGene = {
   type: new GraphQLList(SuggestedGeneType),
-  description: "List of curated genes wit custom images",
+  description: "List of curated genes with custom images",
```

Comment:
Seems odd to me that a "List of curated genes" would be singular "SuggestedGene", but I assume there's a good reason for that?

Reply:
yeah,  the terminology is a bit weird

```js
const SUGGESTED_GENES_JSON = "https://s3.amazonaws.com/eigen-production/json/eigen_categories.json"
const SuggestedGene = {
  type: new GraphQLList(SuggestedGeneType),
  description: "List of curated genes with custom images",
  resolve: () => fetch(SUGGESTED_GENES_JSON).then(({ body }) => body),
}
```

this should be `const SuggestedGenes = {` as that is the definition of the function. The type of `SuggestedGene`  is fine

```
const SuggestedGeneType = new GraphQLObjectType({
  name: "SuggestedGene",
  fields: {
    id: { type: GraphQLString },
    image_url: { type: GraphQLString },
    _id: { type: GraphQLString },
    name: { type: GraphQLString },
  },
})
```